01week/rockPaperScissors.js

```diff
@@ -7,23 +39,61 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
-
+  // write code to remove spaces and case sensitivity
+  hand1 = hand1.toLowerCase().trim();
```

Comment:
It is best practice not to change the given inputs in javascript. Best practice would be to assign to a new variable and evaluate that
```javascript
const  hand1Formatted = hand1.toLowerCase().trim();
```